docs/podman-container.1.md

```diff
@@ -24,8 +24,8 @@ The container command allows you to manage containers
 | export   | [podman-export(1)](podman-export.1.md)              | Export a container's filesystem contents as a tar archive.                   |
 | inspect  | [podman-inspect(1)](podman-inspect.1.md)            | Display a container or image's configuration.                                |
 | kill     | [podman-kill(1)](podman-kill.1.md)                  | Kill the main process in one or more containers.                             |
+| list, ls, ps | [podman-ps(1)](podman-ps.1.md)                 | List the containers on the system.                                     |
```

Comment:
I just wonder if it make sense to have a separate entry for each of these, at least one for ps.  The values in the 2nd and third column could stay the same, it would just be for searching.  i.e. if you'd 'ps' in your noggin, you might not find it under list.

Reply:
Switched to one per line.